crates/burn-import/src/onnx/op_configuration.rs

```diff
@@ -781,19 +792,40 @@ pub fn pad_config(node: &Node) -> PadConfig {
             _ => panic!("Pad: Only tensor input is valid"),
         };
 
-        let pads: Vec<usize> = match &node.inputs[1].value {
-            Some(Data::Int64s(shape)) => shape
-                .iter()
-                .map(|&x| {
-                    if x < 0 {
-                        // TODO: support negative pads
-                        panic!("Pad: Negative pad is not supported");
+        //TODO : handle more possible attributes
+        let mut pads: Vec<usize> = get_input_values(node, 1)
+            .into_iter()
+            .map(|x| x as usize)
+            .collect();
+
+        for (key, value) in node.attrs.iter() {
+            match key.as_str() {
+                "pads" => {
+                    pads = value
+                        .clone()
+                        .into_i64s()
+                        .iter()
+                        .map(|&x| {
+                            if x < 0 {
+                                panic!("Pad: Negative pad is not supported");
+                            }
+                            x as usize
+                        })
+                        .collect()
```

Comment:
If we are supporting the `pads` as attribute, which corresponds to [version 2](https://github.com/onnx/onnx/blob/main/docs/Changelog.md#Pad-2) of the operator, then we should also parse the `value` attribute to make sure we are correctly capturing all parameters of the node.

Make sure it doesn't come in conflict with the `constant_value` input parsing for later versions of the operator 🙂